monitor_frontend/views/events_table.view.lkml

```diff
@@ -0,0 +1,111 @@
+include: "//looker-hub/monitor_frontend/views/events.view.lkml"
+
+view: +events {
+  # # You can specify the table name if it's different from the view name:
+  # sql_table_name: my_schema_name.tester ;;
+  #
+  # # Define your dimensions and measures here, like this:
+  dimension: user_id {
```

Comment:
To follow the convention of the underlying view I'd recommend prefixing these dimensions with `event_extra__`.
```suggestion
  dimension: event_extra__user_id {
```